core/src/main/java/amino/run/kernel/common/ServerInfo.java

```diff
@@ -74,4 +79,47 @@ private boolean matchRequirements(List<Requirement> requirements) {
         }
         return !requirements.isEmpty();
     }
+
+    private void writeObject(java.io.ObjectOutputStream s) throws java.io.IOException {
```

Comment:
I don't understand why this method is here and so weird.  Please comment to explain why you didn't do the standard stuff:

https://www.baeldung.com/java-serialization

Reply:
Added header and comments in it.

Reply:
No you didn't.